rest-service/manager_rest/test/endpoints/test_deployment_updates.py

```diff
@@ -276,33 +277,34 @@ def test_one_active_update_per_deployment_force_flag(self):
         self.assertEquals(STATES.FAILED, first_update.state)
 
     def test_one_active_update_per_dep_force_flag_real_active_executions(self):
-        deployment_id = 'dep'
-        self._deploy_base(deployment_id, 'no_output.yaml')
-        response = self._update(deployment_id, 'one_output.yaml')
-        first_update_id = response.json['id']
-        first_execution_id = response.json['execution_id']
-
-        # updating the execution's status to started to make the first update
-        # really be active
-        execution = self.sm.get(models.Execution, first_execution_id)
-        execution.status = ExecutionState.STARTED
-        self.sm.update(execution)
-        self.client.executions.get(execution_id=first_execution_id)
-
-        response = self._update(deployment_id,
-                                blueprint_name='one_output.yaml',
-                                force=True)
-        # force flag is expected not to work because the first update is
-        # still really running
-        self.assertEquals(response.json['error_code'], 'conflict_error')
-        self.assertIn('there are deployment updates still active',
-                      response.json['message'])
-        self.assertIn('the "force" flag was used',
-                      response.json['message'])
-        # verifying the first update wasn't set with a failed state by the
-        # force flag call
-        first_update = self.client.deployment_updates.get(first_update_id)
-        self.assertEquals(STATES.EXECUTING_WORKFLOW, first_update.state)
+        with validate_execution_transitions():
```

Comment:
Change name to dont validate or validate all or skip validation or maybe you shouldn't really check transition validation please sir goodday